src/main/scala/io/iohk/ethereum/domain/Address.scala

```diff
@@ -19,10 +22,14 @@ case class Address(bytes: ByteString) {
 
   import Address.Length
 
+  //FIXME: can we get rid off it and simply make sure that address is always 20 bytes, by either truncating or extending
```

Comment:
I'd do that right now. It would simplify code in `SignedTransaction`. However it's up to @rtkaczyk :)